images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -42,13 +42,16 @@ Describe "DACFx" {
     It "DACFx" {
         (Get-ItemProperty HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\*).DisplayName -Contains "Microsoft SQL Server Data-Tier Application Framework (x64)" | Should -BeTrue
         $sqlPackagePath = 'C:\Program Files\Microsoft SQL Server\150\DAC\bin\SqlPackage.exe'
-        $sqlLocalDBPath = 'C:\Program Files\Microsoft SQL Server\130\Tools\Binn\SqlLocalDB.exe'
         "${sqlPackagePath}" | Should -Exist
+    }
+
+    It "SqlLocalDB" -Skip:(Test-IsWin22) {
+        $sqlLocalDBPath = 'C:\Program Files\Microsoft SQL Server\130\Tools\Binn\SqlLocalDB.exe'
         "${sqlLocalDBPath}" | Should -Exist
     }
 }
 
-Describe "DotnetTLS" {
+Describe "DotnetTLS" -Skip:(Test-IsWin22) {
```

Comment:
Why do we skip it for the Windows 2022? As I can see, we added the Update-DotnetTLS.ps1 script to the windows2022 template.

Reply:
I think that `Update-DotnetTLS.ps1` and `DotnetTLS` test are not connected directly.
ps1 script sets default TLS version for .NET Core (and it is still needed on Windows Server 2022) but `DotnetTLS` validates system TLS.

System TLS should be set to 1.2 on Windows Server 2016 and 2019. On Windows Server 2022 it is not needed because it uses TLS 1.3 by default (and this test doesn't work properly for it)